src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -119,6 +120,12 @@ public IDataSourceFile SetDataSource(string dataSourceFileName)
          if (string.IsNullOrEmpty(dataSourceFileName)) return null;
          Sheets = new Cache<string, DataSheet>();
          _dataSourceFile = _importer.LoadFile(_columnInfos, dataSourceFileName, _metaDataCategories);
+
+         if (_dataSourceFile == null)
```

Comment:
there is 
{
return null 
}

but the ImporterPresenter uses no { }  in the same PR
Let's be consistent and remove the { }

so 
```
if (_dataSourceFile == null)
  return null
```

Reply:
fixed